views/partials/header.pug

```diff
@@ -2,11 +2,11 @@ header(x-data="{ open: false }")
   nav(class='flex flex-row items-center justify-between px-6 py-4 text-xl')
     a(href='/' class='text-xl font-bold nav-link') TanulóSCH
     div(class='flex flex-row items-center space-x-6')
-      a(onclick='changeTheme()' class='cursor-pointer nav-link md:hidden')
+      a(onclick='changeTheme()' aria-label="Sötét Mód" class='cursor-pointer nav-link md:hidden')
```

Comment:
We should use the same label consistently, I think `Sötét mód váltása` (with only the first word capitalized) would be better like in line 31.